internal/manifests/var.go

```diff
@@ -131,10 +141,38 @@ func serviceNameQueryFrontendHTTP(stackName string) string {
 	return fmt.Sprintf("loki-query-frontend-http-%s", stackName)
 }
 
+func signingServiceSecretName(stackName string) string {
+	return fmt.Sprintf("%s-%s", stackName, "metrics")
+}
+
 func fqdn(serviceName, namespace string) string {
 	return fmt.Sprintf("%s.%s.svc.cluster.local", serviceName, namespace)
 }
 
+func volumeMounts(podSpec corev1.PodSpec, stackName string) corev1.podSpec {
```

Comment:
:x: The function's name is definetly not representative of the work that it accomplishes namely mounting volumes, extending container arguments.

:x: In addition the function is dedicated to patch the pod spec for our service secret, however it does not register the volumes which get mounted to the pod spec.